src/ua.c

```diff
@@ -860,6 +886,22 @@ static const char *autoans_header_name(enum answer_method met)
 }
 
 
+static int ua_check_dst_ip(struct ua *ua, struct pl *pl)
+{
+	struct sip_addr addr;
+	int err;
+
+	sa_init(&ua->dst, AF_UNSPEC);
+	err = sip_addr_decode(&addr, pl);
+	err |= sa_set(&ua->dst, &addr.uri.host, addr.uri.port);
+
+	if (!sa_isset(&ua->dst, SA_PORT))
+		sa_set_port(&ua->dst, 5060);
```

Comment:
port 5060 is hardcoded ...

Reply:
same here.

Reply:
`ua->dst` is also used as destination address to select `laddr` for the RTP traffic. But for outgoing calls we don't have an RTP remote port here.

When we want to use SIP default ports 5060, 5061, ...: We don't have the transport protocol at this point. It is chosen in request.c function `sip_request()` called indirectly via `call_connect()`. But we need  the destination address in `ua_call_alloc()`. Thus which port should we use here?